src/components/TradeWidget/TxMessage.tsx

```diff
@@ -107,16 +107,20 @@ export const SimpleDisplayPrice: React.FC<SimpleDisplayPriceProps> = ({
 }
 
 const Warning = styled.p`
-  background: beige;
-  padding: 0.5em;
-  border-radius: 0.3em;
+  position: relative;
   display: flex;
-  border: 1px solid gray;
-  font-weight: bold;
+  background: var(--color-background-deleteOrders);
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 0.3em;
+
+  > span {
+    width: 94%;
+  }
 
   .alert {
-    min-width: 2.5em;
-    align-self: center;
```

Comment:
Without these my alert :warning:  looks squished
![pr1310--dexreact review gnosisdev com_trade_DAI-USDC_sell=1 price=1 00619 from= expires=](https://user-images.githubusercontent.com/5121491/89871441-0f975080-dbc0-11ea-8760-48de5357e13b.png)

Reply:
You can use `object-fit: contain;` if you don't like `align-self`